backstory/src/test/java/com/google/sps/images/AnnotatedImageTest.java

```diff
@@ -167,12 +191,59 @@ public void getLabelDescriptionsNonEmptyLabelAnnotations() throws IllegalArgumen
       throw exception;
     }
 
-    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations);
+    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations, emptyLandmarkAnnotations);
     List<String> actualLabelDescriptions = annotatedImageActual.getLabelDescriptions();
     List<String> expectedLabelDescriptions = new ArrayList<>();
-    expectedLabelDescriptions.add("DescriptionOne");
-    expectedLabelDescriptions.add("DescriptionTwo");
+    expectedLabelDescriptions.add(firstAnnotationDescription);
+    expectedLabelDescriptions.add(secondAnnotationDescription);
 
     assertEquals(expectedLabelDescriptions, actualLabelDescriptions);
   }
+
+  /**
+   * public List<String> getLandmarkDescriptions();
+   *  landmarkAnnotations is empty.
+  */
+  @Test
+  public void getLandmarkDescriptionsEmptyLandmarkAnnotations() throws IllegalArgumentException, IOException {
+    byte[] rawImageData;
+    try {
+      rawImageData = getBytesFromImageReference(
+        "src/test/java/com/google/sps/images/data/dogRunningOnBeach.jpg", "jpg");
+    } catch (IOException exception) {
+      throw exception;
+    }
+        
+    AnnotatedImage annotatedImage = new AnnotatedImage(rawImageData, emptyLabelAnnotations, emptyLandmarkAnnotations);
+    assertEquals(new ArrayList<String>(), annotatedImage.getLandmarkDescriptions());
+  }
+
+  /**
+   * public List<String> getLandmarkDescriptions();
+   *  landmarkAnnotations is a non-empty list of EntityAnnotations.
+  */
+  @Test
+  public void getLandmarkDescriptionsNonEmptyLandmarkAnnotations() throws IllegalArgumentException, IOException {
+    List<EntityAnnotation> landmarkAnnotations = new ArrayList<EntityAnnotation>();
+    String firstAnnotationDescription = "DescriptionOne";
+    String secondAnnotationDescription = "DescriptionTwo";
+    landmarkAnnotations.add(makeMockEntityAnnotationWithDescription(firstAnnotationDescription));
+    landmarkAnnotations.add(makeMockEntityAnnotationWithDescription(secondAnnotationDescription)); 
+    
+    byte[] rawImageData;
+    try {
+      rawImageData = getBytesFromImageReference(
+        "src/test/java/com/google/sps/images/data/dogRunningOnBeach.jpg", "jpg");
+    } catch (IOException exception) {
+      throw exception;
+    }
```

Comment:
It doesn't seem like try/catch is needed here, since the method declares that it throws IOException.

Reply:
I agree. I'll remove it.

Reply:
Done